lang/en/lang.php

```diff
@@ -93,6 +93,7 @@
             'expand-all' => 'Expand all',
             'loading' => 'Loading',
             'open-in-editor' => 'Open in editor',
+            'no-available-blocks' => 'No available blocks for this editor',
```

Comment:
Maybe this should read
```suggestion
            'no-available-blocks' => 'No blocks available for this editor',
```